### PR TITLE
feat(default-layout) Deprecating the default layout in the documentation

### PR DESCRIPTION
**Pull request by DumitruCorini**
Covers [BPO-116](https://bonitasoft.atlassian.net/browse/BPO-116)

### CODE REVIEW

**Comment by tnguyen1** [comment]
add to whitelist

File: md/taxonomy.md [hunk 1]
```diff
@@ -249,6 +249,8 @@
     * [Connectors execution](connectors-execution.md)
   * [Living application](_living-application.md)
     * [Customize living application layout](customize-living-application-layout.md)
+    * [Customize resource layouts](customize-resource-layouts.md)
+    * [Bonita Layout](bonita-layout.md)
```
Comment (abirembaut) [inline md/taxonomy.md:251]: Maybe Bonita Layout should be the first page in this section

File: md/bonita-layout.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Bonita Layout
```
Comment (abirembaut) [inline md/bonita-layout.md:1]: I think this page needs a sentence explaining what is Bonita layout

File: md/customize-living-application-layout.md [hunk 3]
```diff
@@ -7,89 +7,12 @@
 ::: warning
```
Comment (abirembaut) [inline md/customize-living-application-layout.md]: I think this page could be renamed Living application layout and should contain a short sentence explaining what is living application layout

File: md/customize-resource-layouts.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
+# Customize the resources layouts
```
Comment (abirembaut) [inline md/customize-resource-layouts.md]: customize application layouts seems like a better name to me

File: md/customize-resource-layouts.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+::: info
+**Note:** For Enterprise, Performance, Efficiency, and Teamwork editions only.
+:::
+
+## Overview
+
+As explained in the [layouts](layouts.md) documentation page, it is possible to customize the existing application layout with the UI Designer.
```
Comment (abirembaut) [inline md/customize-resource-layouts.md]: using the UI Designer

File: md/customize-resource-layouts.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+* Import the layout into the UI Designer
+* Modify the layout
+* Apply the customized layout onto a living application.
+
+## Export a layout from Bonita Portal
+
+1. Open Bonita Portal, and log in with username and password of someone mapped to the Administrator profile.
+2. Click on the **Resources** menu.
+3. Select the filter **Layouts**.
+4. Select the element that you want to modify.
```
Comment (abirembaut) [inline md/customize-resource-layouts.md]: select the layout you want to modify

File: md/customize-resource-layouts.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+4. Add the new _myCustomLayout_.
+5. Click **Next**.
+6. Click **Confirm**.
+
+## Configure your living application to use the modified layout
+
+1. Click on the **Applications** menu.
+2. Click on the edit action **...** of your living application.
+3. In the **Look & Feel** section, click on the **Layout** link
+4. Select the _myCustomLayout_
```
Comment (abirembaut) [inline md/customize-resource-layouts.md]: Select _myCustomLayout_
or
Select the layout _myCustomLayout_ layout

**Review by abirembaut — CHANGES_REQUESTED** [review]
just a few comments

File: md/taxonomy.md (hunk 1 above)
Comment (DumitruCorini) [reply md/taxonomy.md:251]: done

File: md/customize-living-application-layout.md (hunk 3 above)
Comment (DumitruCorini) [reply md/customize-living-application-layout.md]: done

File: md/customize-resource-layouts.md (hunk 5 above)
Comment (DumitruCorini) [reply md/customize-resource-layouts.md]: done

File: md/customize-resource-layouts.md (hunk 7 above)
Comment (DumitruCorini) [reply md/customize-resource-layouts.md]: done

File: md/bonita-layout.md (hunk 2 above)
Comment (DumitruCorini) [reply md/bonita-layout.md:1]: done

File: md/customize-resource-layouts.md (hunk 4 above)
Comment (DumitruCorini) [reply md/customize-resource-layouts.md]: done

File: md/customize-resource-layouts.md (hunk 6 above)
Comment (DumitruCorini) [reply md/customize-resource-layouts.md]: done

File: md/bonita-layout.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+::: info
+**Note:** For Enterprise, Performance, Efficiency, and Teamwork editions only.
+:::
+
+This layout has different desktop and mobile versions.
+For the dektop version, the layout contains two rows.
```
Comment (abirembaut) [inline md/bonita-layout.md]: desktop

File: md/bonita-layout.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+The first row has the logo, the name of the application on the left, as well as the user name, which, when clicked, opens the information about the current session. The last button of the first row opens a window that will let you switch between living applications.
+The second row contains the names of the pages.
```
Comment (abirembaut) [inline md/bonita-layout.md]: rather "contains the menu of the application pages" ?

File: md/bonita-layout.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+For the mobile version there is only one row that has the name of the application, as well as a button that opens a dropdown. The dropdown will contain the names of the pages, the user name which, when clicked opens the information about the current session, as well as the button that opens the navigation between applications.
```
Comment (abirembaut) [inline md/bonita-layout.md]: The dropdown will contain the menu of application pages

File: md/bonita-layout.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+## Customizing this layout
+
+You can customize this layout by following the [customize-resource-layouts](customize-resource-layouts.md) steps.
+
+## Adding a logo to the layout
+
+The first option to change the theme that is used by the application is to add a "logo.png" file in the "resources/images/" folder. If the folder does not exist, create it manually.
```
Comment (abirembaut) [inline md/bonita-layout.md]: The first option is changing the theme that is used by the application by adding a "logo.png

File: md/bonita-layout.md [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+The second option is adding an image asset to the layout in UI Designer, and using the asset name.
```
Comment (abirembaut) [inline md/bonita-layout.md]: in the UI Designer, and using this asset directly in the URL property of the image widget.

**Review by abirembaut — COMMENTED** [review]
Just a few more comments

File: md/bonita-layout.md (hunk 8 above)
Comment (DumitruCorini) [reply md/bonita-layout.md]: done

File: md/bonita-layout.md (hunk 9 above)
Comment (DumitruCorini) [reply md/bonita-layout.md]: done

File: md/bonita-layout.md (hunk 11 above)
Comment (DumitruCorini) [reply md/bonita-layout.md]: done

File: md/bonita-layout.md (hunk 12 above)
Comment (DumitruCorini) [reply md/bonita-layout.md]: done

File: md/bonita-layout.md (hunk 10 above)
Comment (DumitruCorini) [reply md/bonita-layout.md]: done

**Review by abirembaut — APPROVED** [review]

**Pull request merged**